src/main/java/io/nats/client/support/JwtUtils.java

```diff
@@ -0,0 +1,145 @@
+// Copyright 2021 The NATS Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package io.nats.client.support;
+
+import io.nats.client.NKey;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.security.GeneralSecurityException;
+import java.security.MessageDigest;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.Base64;
+
+import static io.nats.client.support.JsonUtils.beginJson;
+import static io.nats.client.support.JsonUtils.endJson;
+
+public class JwtUtils {
+    static class Nats implements JsonSerializable {
+        String issuerAccount;
+        String[] tags;
+
+        @Override
+        public String toJson() {
+            StringBuilder sb = beginJson();
+            JsonUtils.addField(sb, "issuer_account", issuerAccount);
+            JsonUtils.addStrings(sb, "tags", Arrays.asList(tags));
```

Comment:
Note that all of these `JsonUtils` are not properly handling JSON string encoding for characters that need to be escaped. For example `"`, `\` or any control character such as `\n`. I would recommend asserting that all `String` inputs to APIs validate that these special characters are not found... or properly encode strings. Without this, a tag with a Windows fileshare (for example) would cause an invalid JWT to be minted and cause much confusion since no error would occur until the JWT was actually used.

Reply:
yeah the json utils assumed data from the server or from the client, so this type of checking was unnecessary. Not sure how to address this, but it needs addressing

Reply:
Does java have no native object to json serializer?

Reply:
java does not and we've been asked to avoid 3rd party libraries

Reply:
Based on conversation off-line, we can easily fix this by modifying `JsonUtils` so these characters are properly encoded.

Reply:
`JsonUtils` now properly escapes string characters that need escaping per the spec.